src/redux/store.js

```diff
@@ -1,16 +1,30 @@
 import { applyMiddleware, createStore } from 'redux';
 
 import logger from 'redux-logger';
-import promise from 'redux-promise-middleware';
+import root from './reducer';
 
-import userReducer from './reducer.js';
+let initialState = {
```

Comment:
instead of setting this initial state here you should do that in the reducers. This parameters should be used to load any saved data from session or cookies not setting default values for the state.